packages/next/types/global.d.ts

```diff
@@ -22,10 +22,6 @@ declare namespace NodeJS {
     // TODO: Should be optional and possibly undefined
     readonly NODE_ENV: 'development' | 'production' | 'test'
   }
-
-  interface RequestInit extends globalThis.RequestInit {
-    next?: NextFetchRequestConfig | undefined
-  }
```

Comment:
This was a duplicate, which yielded an error after updating eslint. It's also defined further down in this file with global interface augmentation.